player/command.c

```diff
@@ -7270,10 +7275,6 @@ void handle_command_updates(struct MPContext *mpctx)
 
     // Depends on polling demuxer wakeup callback notifications.
     cache_dump_poll(mpctx);
-
-    // Potentially run the commands now (idle) instead of waiting for a file to load.
-    if (mpctx->stop_play == PT_STOP)
```

Comment:
I think `PT_STOP` is more powerful than `MPV_EVENT_IDLE`. I don't mind the change, but we not always transition to idle, iirc. So this changes behavior.

Reply:
It was open question. I'm not using `input-commands`, hard to tell for me when they should run preferably. I'll defer to @Dudemanguy judgment.

Reply:
`MPV_EVENT_IDLE` should be fine.